rails/commands-pattern/app/commands/users/create.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+# Notice that commands are scoped into a module referencing the entity they are working
+# with. There are situations where the command might not refer to a single entity.
+# The name of the module doesn't need to match the name of a Rails model.
+module Users
+  class Create
```

Comment:
I think we should use a suffix in the class name like `CreateCommand`, In Rails all classes except models have a suffix and other gems like Keynote uses prefixes in its classes too. Use suffixes could be useful for the project's newbies too.

We could think we only should use suffix if the class have an inheritance but in the Form Objects patter we use Object without inheritance and its classes have suffix too.